### PR TITLE
[SMALLFIX] Quit KVTest if kv not enabled, with error message

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

**Comment by apc999** [comment]
alluxio-bot, check this please

File: examples/src/main/java/alluxio/examples/keyvalue/KeyValueStoreOperations.java [hunk 1]
```diff
@@ -131,6 +131,12 @@ public static void main(String[] args) throws Exception {
       System.exit(-1);
     }
 
+    if (!Configuration.getBoolean(Constants.KEY_VALUE_ENABLED)) {
+      System.out.println("Alluxio key value service is disabled. To run this test, please set "
+          + Constants.KEY_VALUE_ENABLED + " to be true, restart the cluster");
```
Comment (gpang) [inline examples/src/main/java/alluxio/examples/keyvalue/KeyValueStoreOperations.java]: NIT: "and" restart the cluster

**Comment by gpang** [comment]
@apc999 Thanks. I left a minor comment.

**Comment by apc999** [comment]
@gpang fixed. PTAL. thanks

**Comment by uronce-cc** [comment]
LGTM

**Pull request merged**